doc/stdlib/file.rst

```diff
@@ -337,7 +337,7 @@
 
    .. Docstring generated from Julia source
 
-   ``@__FILE__`` expands to a string with the absolute file path of the file containing the macro. Returns ``nothing`` if run from a REPL or an empty string if evaluated by ``julia -e <expr>``. Alternatively see :data:`PROGRAM_FILE`.
+   ``@__FILE__`` expands to a string with the absolute file path of the file containing the macro. Returns ``nothing`` if run from a REPL or an empty string if evaluated by ``julia -e <expr>``\ . Alternatively see ```PROGRAM_FILE`` <:data:`PROGRAM_FILE`>`_\ .
```

Comment:
At the moment we have to special case the rendering of those, see https://github.com/JuliaLang/julia/blob/4d1bf92752a3efb6370980b722d8832d632ec9fd/base/markdown/render/rst.jl#L86. Just needs `data` added to the regex I think.